browser/resources/settings/shortcuts_page/components/Keys.tsx

```diff
@@ -39,7 +39,7 @@ const Kbd = styled.div<{ large?: boolean; square?: boolean }>`
       min-width: 32px;
       border-radius: ${radius[12]};
       padding: ${spacing[16]};
-      font: var(--leo-font-heading-h3);
+      font: var(--leo-font-primary-heading-h3);
```

Comment:
@petemill I wanted to use the Leo tokens instead of this, but they're a bit broken:

`font.primary.heading.h3` does not exist but `font.desktop.primary.heading.h3` does. Unfortunately it points to `var(--leo-font-desktop-primary-heading-h3)`, which does not exist (I think we strip the `desktop`).

Reply:
https://github.com/brave/leo/pull/307 has a fix for this. I'm thinking it might be worth waiting until that lands to bump.